tests/publiccloud/boottime.pm

```diff
@@ -308,11 +308,11 @@ sub measure_timings {
         $provider = $args->{my_provider};
     } else {
         $provider = $self->provider_factory();
-        $instance = $self->{my_instance} = $provider->create_instance(check_connectivity => 0);
+        $instance = $self->{my_instance} = $provider->create_instance();
     }
 
-    $ret->{analyze}->{ssh_access} = $instance->wait_for_ssh(timeout => 300);
-    assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
+    #$ret->{analyze}->{ssh_access} = $instance->wait_for_ssh(timeout => 300);
+    #assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
```

Comment:
why you commented this one ?

Reply:
these 2 commented lines are already executed in create_instance, infact I also added to its call the parameter to enable internal wait_ssh

Reply:
Removed.